featuretools/entityset/deserialize.py

```diff
@@ -99,7 +101,12 @@ def empty_dataframe(description):
             cat_dtype = col["physical_type"]["cat_dtype"]
             cat_object = pd.CategoricalDtype(pd.Index(cat_values, dtype=cat_dtype))
             category_dtypes[col_name] = cat_object
+
     dataframe = pd.DataFrame(columns=columns).astype(category_dtypes)
+    if data_type == "Dask":
+        dataframe = dd.from_pandas(dataframe, npartitions=1)
+    elif data_type == "Spark":
```

Comment:
Isn't the point of Enums so that you can do this? 
```suggestion
    if data_type == Library.DASK:
        dataframe = dd.from_pandas(dataframe, npartitions=1)
    elif data_type == Library.SPARK:
```